installer/manifests/keptn/charts/control-plane/templates/mongodb-credentials.yaml

```diff
@@ -27,6 +32,10 @@
 {{- $mongoAdminPassword = index $mongosecret.data "admin_password" -}}
 {{- end -}}
 
+{{- if $mongosecret -}}
+{{- $mongoExternalConnectionString = index $mongosecret.data "external_connection_string" -}}
```

Comment:
I would add a check here to see if the value is really set in the secret, before referencing it. Somthing like `{{ if index $mongosecret.data "external_connection_string" }}`